app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -232,15 +253,24 @@ class RefereeProfile extends Component {
   }
 
   renderCertificationContent = () => {
-    const { referee: { certifications, isEditable, submittedPaymentAt } } = this.state
+    const {
+      referee: {
+        certifications, isEditable, submittedPaymentAt, nationalGoverningBodies, testResults, testAttempts
+      }
+    } = this.state
     const { match: { params } } = this.props
 
+    const shouldTakeOldTests = nationalGoverningBodies.some(ngb => /australia/i.test(ngb.name))
```

Comment:
Please add Switzerland

Reply:
Also: Would it be able to show referees from these NGBs a second tab with the old tests so they are able to also be certified in the new ones (for tournaments from other NGBs) or at least a profile option if they like to do the old or the new ones? Do we know if they’re qualified for the new or the old tests?

Reply:
I'm not sure on that, the attitude I gather is that Australia specifically asked just to take the old tests. It's pretty easy to update so for the time being will leave as-is until I can get some clarity from RDT.